MacOS/Python/Install.sh

```diff
@@ -65,24 +65,55 @@ _py_version=$PYTHON_VERSION_PS
 echo "Checking for Miniconda or Anaconda installation..."
 if conda --version > /dev/null; then
     echo "Miniconda or Anaconda is already installed."
-    # print conda version 
-    echo "Conda version: $(conda --version)"
-    clear -x 
-    echo "To proceed with the installation, the existing Conda installation must be removed."
-    read -p "Do you want to uninstall it? (Type 'yes' to confirm): " confirm
-    if [ "$confirm" = "yes" ]; then
-        echo "Uninstalling existing Conda installation..."
-        conda init --reverse --all
-        cd
-        rm -rf ~/anaconda3
-        rm -rf ~/miniconda3
-        sudo rm -rf /opt/anaconda3
-        sudo rm -rf /opt/miniconda3
-        echo "Existing Conda installation has been removed."
-    else
-        echo "Installation aborted. The script cannot proceed without removing the existing Conda installation."
-        exit 0
+
+# checks for homebrew installations 
+    if [[ $(conda info --base 2> /dev/null) == *"/Caskroom/miniconda"* ]]; then
+        echo "Existing Miniconda installation through homebrew detected."
+        echo "Do you want to uninstall it?"
+        if [ "$confirm" = "yes" ]; then
```

Comment:
where is `confirm` read?

Reply:
Adressed in commit https://github.com/dtudk/pythonsupport-scripts/pull/51/commits/da5e210fa3d52a6f5550be6e67eae02521d9f0ab